client/trino-client/src/main/java/io/trino/client/ClientInfoProperty.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.trino.jdbc;
+package io.trino.client;
```

Comment:
this was essentially duplicated in the jdbc package. extracting it out so i can use it i the adbc package